### PR TITLE
Fix broken links and minor updates to contributing guide and installation instructions

### LINKED ISSUE #56: Update contributing guidelines and remove references to old GeoCAT guidelines

### LINKED ISSUE #59: Update installation instructions
Still referencing the old conda channel - yikes.

### PR DESCRIPTION
**Pull request by kafitzgerald**
Fix broken links and minor updates to contributing guide and installation instructions

Closes #59 
Closes #56

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
-Please first refer to [GeoCAT Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) for overall 
-contribution guidelines (such as detailed description of GeoCAT structure, forking, repository cloning, 
-branching, etc.). Once you determine that a data file should be added to this repository, 
-please refer to the following contribution guidelines:
+Please refer to the [GeoCAT website](https://geocat.ucar.edu/) for an overview of the broader GeoCAT project
+and relevant resources.
```
Comment (philipc2) [inline CONTRIBUTING.md:2]:
```suggestion
Please refer to the [GeoCAT website](https://geocat.ucar.edu/) for an overview of the GeoCAT project
and relevant resources.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 
+Below are some guidelines for adding new files to geocat-datafiles.
```
Comment (philipc2) [inline CONTRIBUTING.md:4]:
```suggestion
Below are guidelines for adding new files to geocat-datafiles:
```

File: INSTALLATION.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
 # Installation
 
-This installation guide includes only the GeoCAT-datafiles installation instructions. 
-Please refer to [GeoCAT Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) for installation of 
-the whole GeoCAT project.
+This installation guide includes instructions only for GeoCAT-datafiles.
```
Comment (philipc2) [inline INSTALLATION.md:3]:
```suggestion
This installation guide includes instructions only for geocat-datafiles. 
```

Should this be lowercase for consistency with the other parts?

File: INSTALLATION.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
...
 
-This installation guide includes only the GeoCAT-datafiles installation instructions. 
-Please refer to [GeoCAT Contributor's Guide](https://geocat.ucar.edu/pages/contributing.html) for installation of 
-the whole GeoCAT project.
+This installation guide includes instructions only for GeoCAT-datafiles. 
+Please refer to the [GeoCAT website](https://geocat.ucar.edu/pages/software.html) for links to resources for other GeoCAT projects.
   
 
-## Installing GeoCAT-datafiles via Conda in a New Environment
+## Installing GeoCAT-datafiles via Conda in a new environment
```
Comment (philipc2) [inline INSTALLATION.md:7]: Same comment regarding "geocat-datafiles"

File: INSTALLATION.md [hunk 5]
```diff
@@ -1,29 +1,29 @@
...
 activated using:
 
     conda activate geocat
 
-If you somewhat need to make use of other software packages, such as Matplotlib, Cartopy, Jupyter, etc. with
-GeoCAT-datafiles, you may wish to install into your `geocat` environment. The following `conda create` command can be
-used to create a new conda environment that includes some of these additional commonly used Python packages pre-installed:
+If you would like to make use of other software packages, such as Matplotlib, Cartopy, Jupyter, etc. with
+GeoCAT-datafiles, you may wish to install these into your `geocat` environment as well. The following
+command can be used to create a new conda environment that includes some of these additional Python packages:
```
Comment (philipc2) [inline INSTALLATION.md:20]:
```suggestion
command can be run to create a new conda environment that includes some of these additional Python packages:
```

**Review by philipc2 — CHANGES_REQUESTED** [review]
Looks great! A few slight phrasing and consistency suggestions, but nothing major.

**Comment by kafitzgerald** [comment]
> Looks great! A few slight phrasing and consistency suggestions, but nothing major.

Thanks!  These look great.

**Review by philipc2 — APPROVED** [review]
Great additions!

File: INSTALLATION.md (hunk 4 above)
Comment (kafitzgerald) [reply INSTALLATION.md:7]: Totally, and FWIW I'm open to approaching this differently (especially if we have consistent practices across content), but this seems better than it was at least 🙃 .

**Pull request merged**